inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sink/iceberg/IcebergSinkRequest.java

```diff
@@ -64,4 +64,7 @@ public class IcebergSinkRequest extends SinkRequest {
     @ApiModelProperty("Primary key")
     private String primaryKey;
 
+    @ApiModelProperty("append mode, UPSERT or APPEND")
```

Comment:
Do we need to add a pattern to limit the value of this field?

Such as:
```java
    @ApiModelProperty("Append mode, UPSERT or APPEND")
    @Pattern(regexp = "(?i)(UPSERT|APPEND)", message = "Invalid append mode")
```

Reply:
fixed, thx